doc/langref.html.in

```diff
@@ -4489,8 +4489,10 @@ comptime {
       <pre>{#syntax#}@cmpxchgStrong(comptime T: type, ptr: *T, expected_value: T, new_value: T, success_order: AtomicOrder, fail_order: AtomicOrder) ?T{#endsyntax#}</pre>
       <p>
       This function performs a strong atomic compare-and-exchange operation, returning {#syntax#}null{#endsyntax#}
-      if the current value is not the given expected value. It's the equivalent of this code,
-      except atomic:
+      if the value at {#syntax#}ptr{#endsyntax#} was equal to {#syntax#}expected_value{#endsyntax#}
+      at the time of the operation and therefore changed to {#syntax#}new_value{#endsyntax#};
```

Comment:
There's no `then` clause to this leading `if` statement, which makes this confusing to read. What about something like "_If the ptr was equal to expected_value and thus changed to new_value, then null is returned. Otherwise .._"